db/0023-add-division-title-priority.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE `divisions`
+  ADD COLUMN title_priority int(3) default 1;
```

Comment:
Mostly the idea of a priority order was easier with an intenum, which is clear enough if staying in python - but given we'll sometimes access the database in other ways I'll see if that's a quick change.

Reply:
This is https://github.com/mysociety/theyworkforyou/pull/1765/commits/9cb696dfee3debaef25d1f3f16de488fa7a8e740
and https://github.com/mysociety/theyworkforyou/pull/1765/commits/978a6b43bd700219104761ea4be65807bb8af414

Now uses a ENUM in the database, but stores the priorities in the python file. 

This also flushed out I wasn't properly updating the new header - so that's now working properly when it updates things.